jobs_board/core.py

```diff
@@ -1,9 +1,9 @@
 from flask import Flask
 from flask.ext.foundation import Foundation
 from flask.ext.mail import Mail
-from flask.ext.security import Security, SQLAlchemyUserDatastore
+from flask_security import Security, SQLAlchemyUserDatastore
 from flask.ext.sqlalchemy import SQLAlchemy
-
+from jobs_board.forms import ExtendedRegisterForm
```

Comment:
I prefer explicit relative imports here and a newline between third-party and local imports